### PR TITLE
Change the color to toggle in light and dark mode in workflows

### LINKED ISSUE #4260: [NV-2841] Workflow Editor - channels icons are white in light theme
## Reproduction steps
Light theme - channels icon over provider image are always white.
image

Also on hover on the sidebar:
image

Dark theme for reference: 
image

## Definition of done
- RIGHT TOOLBAR. When a user hover over the icons for Email and SMS in light theme it should behaves same way as other channels
- NODE. The channel icons (small icon on the right bottom corner near the provider icon) on the step node should be the following:
  - Dark theme: #BEBECC (B80)
  - Light theme: #828299 (B60)

### PR DESCRIPTION
**Pull request by Aniketsahu28**
### What change does this PR introduce?

This PR fixes channels icon over provider image to appear on light theme as well. It toggles its color to white in dark theme and B60 in light mode.

### Why was this change needed?

The change was to toggle color with respect to the theme. Show white in dark mode and B60 in light mode.
Fixes #4260

Working screenshots
image

### CODE REVIEW

File: apps/web/src/design-system/icons/general/ChatFilled.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ export function ChatFilled(props: React.ComponentPropsWithoutRef<'svg'>) {
       {...props}
     >
       <path
-        fill="url(#paint0_linear_528_4335)"
+        fill={theme.colorScheme === 'light' ? colors.B60 : 'white'}
```
Comment (ainouzgali) [inline apps/web/src/design-system/icons/general/ChatFilled.tsx:16]: @Aniketsahu28 you do not have the theme or the colors import here. Am I missing something?

**Review by ainouzgali — COMMENTED** [review]
@Aniketsahu28 thank you for contributing!! 
Could you please add screenshots of the result? :)

File: apps/web/src/design-system/icons/general/ChatFilled.tsx (hunk 1 above)
Comment (Aniketsahu28) [reply apps/web/src/design-system/icons/general/ChatFilled.tsx:16]: I will make the changes and upload screenshot in a while

Comment (Aniketsahu28) [reply apps/web/src/design-system/icons/general/ChatFilled.tsx:19]: Will you please check  is it fine ?

Comment (rjangid9080) [reply apps/web/src/design-system/icons/general/ChatFilled.tsx:19]: @Aniketsahu28 are you able to run this repo locally ?

Comment (shaikahmadnawaz) [reply apps/web/src/design-system/icons/general/ChatFilled.tsx:19]: > @Aniketsahu28 are you able to run this repo locally ?

I tried 3+ times, but I am not able to run it locally 🥲 @Aniketsahu28 @rjangid9080 @ainouzgali 

Screenshot 2023-10-01 153937

**Comment by scopsy** [comment]
@Aniketsahu28 this issue is not assigned to you, please ask to be assigned before submitting a PR 🙏 In case that the assigned on the issue contributor won't be able to do it and you will be assigned, feel free to reopen this PR.

**Pull request closed without merging**